ci/azure-pipeline-connectors-mysql.yaml

```diff
@@ -4,6 +4,8 @@ trigger:
       - main
       - latest
   paths:
+    exclude:
+      - '*'
```

Comment:
Why was this added?

Reply:
Trying to stop all of the integration tests from running all of the time. This was actually the second attempt, but it doesn't look like either worked. /* is what I had in there before... I can roll back the changes or we could try something else

Reply:
Let's explore this apart from this PR (rollback).

In order to test, we need two PRs. The first one changes the yaml files (which would trigger a rebuild of all tests). The second PR (based on the first one, and targeting the branch of the first PR) only touches files within the Management sample. The second PR should run only the tests for management.